docs/devguides/contribution_guide.rst

```diff
@@ -0,0 +1,92 @@
+
+Contribution Guide
+=====================
+
+We are happy when you share your research with us and when you improve our
+code! There is an easy way to contribute to our project, follow the steps
+below. Your commit will be reviewed and added to our master branch. Moreover,
+if you are a dedicated contributor, you have a chance to get our t-shirt,
+get invited to one of our events or even join our team ; )
+
+How to contribute:
+
+#. Don't start the coding first. You should **post an issue** to discuss the features
+   you want to add. If our team or other contributors accept your offer
+   or give a +1, assign the issue to yourself. Now proceed with coding : )
```

Comment:
We should add something about conforming to pep-8 in everything except line length.

Reply:
resolved earlier